assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js

```diff
@@ -62,6 +64,9 @@ function DashboardClicksWidget() {
 			args.url = url;
 			serviceBaseURLArgs.page = `!${ url }`;
 		}
+		if ( isDomainProperty && referenceSiteURL ) {
```

Comment:
See above, this needs to be an `else if` for the `if ( url )` check. This also applies to all similar cases throughout this PR.